app/templates/main/upload.html

```diff
@@ -93,11 +93,11 @@ <h1 class="govuk-heading-l">Submit monitoring and evaluation data</h1>
                     }) }}
 
                 {% else %}
-                    <h2 class="govuk-heading-m">There are errors in your return</h2>
-                    <p class="govuk-body">The following errors have been found in your return.</p>
+                    <h1 class="govuk-heading-l">There are errors in your return</h1>
```

Comment:
There shouldn't be more than one `<h1>` on a page. If we want the styling, that should be achievable just using the class.

Reply:
I can change this, but it raises a bigger question about wheter we are using this upload.html template to do too much. It has two ```h1```'s to cover the both senarios. Validation error or no error. Potentially there is a refactor that can be done to split these two into seperate templates.